library/private/interactor_impl.h

```diff
@@ -35,15 +35,20 @@ class interactor_impl : public interactor
   interactor_impl(options& options, window_impl& window, scene_impl& scene);
   ~interactor_impl() override;
 
-  interactor& addCommandCallback(
-    std::string action, std::function<bool(const std::vector<std::string>&)> callback) override;
-  interactor& removeCommandCallback(const std::string& action) override;
+  interactor& initializeDefaultCommands() override;
+  interactor& addCommand(const std::string& action,
+    std::function<bool(const std::vector<std::string>&)> callback) override;
+  interactor& removeCommand(const std::string& action) override;
+  std::vector<std::string> getCommandActions() const override;
```

Comment:
why not just `getCommands`?

Reply:
because you dont get access to the callbacks, only the actions.